web/components/admin/ApplicantInfo/index.js

```diff
@@ -36,13 +38,34 @@ class ApplicantInfo extends React.Component {
 
   render() {
     const { className } = this.props;
+    const shortInfo = this.getApplicantShortInfo();
+    const longInfo = this.getApplicantLongInfo();
     return (
       <div className={`applicant-info pad-sides-s pad-ends-s ${className}`}>
         <i>TODO: Applicant info component</i>
-        <h5>Short info</h5>
-        {JSON.stringify(this.getApplicantShortInfo())}
-        <h5>Long info</h5>
-        {JSON.stringify(this.getApplicantLongInfo())}
+        <ShortField label="First name" value={shortInfo ? shortInfo.firstName : ''} />
+        <ShortField label="Last name" value={shortInfo ? shortInfo.lastName : ''} />
+        <ShortField label="Email" value={shortInfo ? shortInfo.email : ''} />
+        <ShortField label="City" value={shortInfo ? shortInfo.city : ''} />
+        <ShortField label="School" value={shortInfo ? shortInfo.school : ''} />
+        <ShortField label="Gender" value={shortInfo ? shortInfo.gender : ''} />
+        <ShortField label="Birthday" value={shortInfo ? shortInfo.birthdate : ''} />
+        <ShortField label="Education" value={shortInfo ? shortInfo.education : ''} />
+        <ShortField label="Grad year" value={shortInfo ? shortInfo.gradYear : ''} />
+        <ShortField label="Travel" value={shortInfo ? shortInfo.travel : ''} />
+        <ShortField label="Major" value={shortInfo ? shortInfo.major : ''} />
+        <ShortField label="Phone Number" value={shortInfo ? shortInfo.phoneNumber : ''} />
+        <ShortField label="Ethnicity" value={shortInfo ? shortInfo.ethnicity : ''} />
+        <ShortField label="First Hackathon?" value={shortInfo ? shortInfo.isFirstHackathon : ''} />
+        <ShortField label="Github" value={shortInfo ? shortInfo.githubLink : ''} />
+        <ShortField label="Personal Website" value={shortInfo ? shortInfo.personalWebsiteLink : ''} />
+        <ShortField label="LinkedIn" value={shortInfo ? shortInfo.linkedInLink : ''} />
+        <ShortField label="Resume" value={shortInfo ? shortInfo.resumeLink : ''} />
```

Comment:
We should hyperlink GitHub, personal website, linkedin, and resume as seen in the mocks. Maybe have an `isUrl` prop for your short field component. It should make it a lot easier for logistics to quickly mark applicants.